WordPress/Classes/ViewRelated/AccountSettingsViewController.swift

```diff
@@ -3,12 +3,14 @@ import UIKit
 import RxSwift
 import WordPressComAnalytics
 
-func AccountSettingsViewController(account account: WPAccount) -> ImmuTableViewController {
-    let service = AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+func AccountSettingsViewController(account account: WPAccount?) -> ImmuTableViewController {
+    let service = account.map({ account in
+        return AccountSettingsService(userID: account.userID.integerValue, api: account.restApi)
+    })
```

Comment:
I'd personally just drop the parenthesis because: although that's a functional call, it's not being chained to anything else.

Nevertheless, it's absolutely your call sir!